src/libs/PolicyUtils.js

```diff
@@ -133,6 +133,13 @@ function getClientPolicyMemberEmailsToAccountIDs(policyMembers, personalDetails)
     return memberEmailsToAccountIDs;
 }
 
+/**
+ * Get login list that we should not show in the workspace invite options
```

Comment:
Now reading this makes me feel this method shouldn't be part `PolicyUtils` as it serves no purpose here.

cc: @pecanoro

Reply:
What was the reasoning to move this here from the component?

Reply:
@pecanoro 
I moved the code because it's independent with the component and it's related policy members

Reply:
I am not against leaving it here but then we need a better name. Like `getEligibleUsersForWorkspace` or something along the lines.

Reply:
I realized my suggestion was the opposite. What about `getIneligibleInvitees`?

Reply:
Why is it ineligible?

Reply:
Because they cannot get invited to the workspace?

Reply:
Or maybe even `getExcludedInvitees`, but I feel that `getExcludedUsers` is vague since it's in PolicyUtils and not in the component.

Reply:
I am also open to suggestions, but what if someone else decides to use that function, like "excluded users in a policy for what"

Reply:
oops, yeah. sorry. fixed